lib/remote-data-umd.js

```diff
@@ -112,6 +112,31 @@ return /******/ (function(modules) { // webpackBootstrap
 	      return config;
 	    }
 	  }, {
+	    key: 'case',
+	    value: function _case(map) {
+	      // Caller must supply a handler for every case OR supply a default handler named _.
+	      if (false) {
```

Comment:
I would actually argue that this file _should_ have the debugging in, and that we should produce a `remote-data-umd.min.js` that is minified and has this stripped out. WDYT?

This would mean in the dev webpack build we'd define `process.env.NODE_ENV` as `development`.